premis_event_service/presentation.py

```diff
@@ -59,30 +73,24 @@ def premisEventXMLToObject(eventXML):
             linkObject = LinkObject.objects.get(
                 object_identifier=identifierValue
             )
-        except LinkObject.DoesNotExist, dne:
+        except LinkObject.DoesNotExist:
             linkObject = LinkObject()
             linkObject.object_identifier = identifierValue
             linkObject.object_type = getValueByName(
                 linkingObjectIDNode, "linkingObjectIdentifierType")
             linkObject.object_role = getValueByName(
                 linkingObjectIDNode, "linkingObjectRole")
             linkObject.save()
-        except LinkObject.MultipleObjectsReturned, mob:
+        except LinkObject.MultipleObjectsReturned:
             linkObject = LinkObject.objects.filter(
                 object_identifier=identifierValue
             )[0]
         newEventObject.linking_objects.add(linkObject)
     datetimeObject = None
-    dateString = newEventObject.event_date_time.split(".", 1)[0]
-    try:
-        datetimeObject = datetime.strptime(dateString, dateFormat)
-    except ValueError:
-        try:
-            datetimeObject = datetime.strptime(dateString, altDateFormat)
-        except ValueError:
-            raise Exception("Unable to parse %s (%s) into datetime object" %
-                            (dateString, newEventObject.event_date_time))
-    newEventObject.event_date_time = datetimeObject
+    if isinstance(newEventObject.event_date_time, basestring):
```

Comment:
Is it possible for `newEventObject.event_date_time` to be a `basestring` here with what is in line 62?